ee/session_recordings/ai/error_clustering.py

```diff
@@ -66,9 +66,9 @@ def cluster_embeddings(embeddings):
     return dbscan.labels_
 
 
-def construct_response(df: pd.DataFrame, team: Team, user: User):
+def construct_response(df: pd.DataFrame, team: Team):
     viewed_session_ids = list(
-        SessionRecordingViewed.objects.filter(team=team, user=user, session_id__in=df["session_id"].unique())
```

Comment:
Dropping the user from viewed counts as the result will be cached once for all teams